test/utils/createIdentityProof.testSuite.ts

```diff
@@ -0,0 +1,29 @@
+import { IAM } from "../../src/iam";
+import { createIam, rootOwner } from "../iam.test";
+import { rpcUrl } from "../setup_contracts";
+
+export const createIdentityProofTests = () => {
+    let rootOwnerIam: IAM;
+
+    beforeAll(async () => {
+        rootOwnerIam = await createIam(rootOwner.privateKey, { createDocument: true });
+    });
+
+    //createIdentityProof Test
+    test("Identity token should be defined", async () => {
+        const identityToken = await rootOwnerIam.createIdentityProof();
+        expect(identityToken).toBeDefined;
+    });
+
+    //createIdentityProofWithDelegate
+    test("Identity token should be defined for delegate", async () => {
```

Comment:
I think that more useful would be scenario when delegate would pass `LoginStrategy` authentication on behalf of identity

Reply:
We finally set this kind of test on passport-did-auth